splunktalib/rest.py

```diff
@@ -66,7 +66,7 @@ def splunkd_request(
                         msg_temp, splunkd_uri, resp.status, code_to_msg(resp, content)
                     )
             else:
-                return resp, content
+                return resp, content.decode("utf-8")
```

Comment:
This is a breaking change, every client of this library should update its code before migrating to the version that will be released by this fix.

I would like not to release anything breaking, especially that we are going to get rid of httplib2 completely and this code will be replaced soon.